.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
if we want to make it faster we can start to cache dependencies as well

Reply:
OK I'll follow https://github.com/symphonyoss/messageml-utils/blob/master/.github/workflows/build.yml#L16

Reply:
👍  done below